modules/efk-logging-kibana-limits.adoc

```diff
@@ -5,18 +5,20 @@
 [id="efk-logging-kibana-limits-{context}"]
 = Configure Kibana CPU and memory limits
 
-Each component specification allows for adjustments to both the CPU and memory limits. 
+Each component specification allows for adjustments to both the CPU and memory limits. You should not need to manually adjust these values as the Cluster Logging Operator sets values sufficient for your environment.
 
 .Prerequisite
 
-If needed, get the name of the Cluster Logging Custom Resource from the `openshift-logging` project:
-
+* If needed, get the name of the Cluster Logging Custom Resource from the `openshift-logging` project:
```

Comment:
When https://github.com/openshift/cluster-logging-operator/pull/123 merges the resource will be named 'instance'